jena-fuseki2/jena-fuseki-docker/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
+
+## env | sort
+exec "$JAVA_HOME/bin/java" $JAVA_OPTIONS -jar ${FUSEKI_DIR}/${FUSEKI_JAR} "$@"
```

Comment:
Maybe ` ${FUSEKI_DIR}/${FUSEKI_JAR}` should be `"${FUSEKI_DIR}/${FUSEKI_JAR}"`? What if there are spaces in the filepaths?

Reply:
Done